installer/sharch_body.sh

```diff
@@ -24,6 +24,10 @@ if [ "$sha1" != "$payload_sha1" ] ; then
 fi
 
 echo " OK."
+#Image size in KB
+bytes=$((($(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) + 1023 ) / 1024))
```

Comment:
see QiLuo's change... no need to converto to KB

Reply:
Below is his comments,
Here it is dividing one time only with 1024, and mouting size in MB
bytes = $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) 
image_size=$((bytes + 1023) / 1024)
mount_size=$((image_size + 31) / 32) * 32
mount -o remount,size="${mount_size}M"